stable/admin/values.yaml

```diff
@@ -82,6 +82,24 @@ admin:
   # Admin Service
   replicas: 1
 
+  # If specified and set to a value greater than 0, an initial membership
```

Comment:
should the default not be 0? That is the current default. Setting it to greater than 1 should solve the -0 issue

Reply:
No, the current value is equivalent to 0 because we never define a bootstrap server ID.

Reply:
if we dont define it, it defaults to -0, correct? Which is logically the same as defining 1. Defining 0 is confusing. It seems to indicate that it is OK to generate separate subgroups.

Reply:
1 and 0 are not the same. 1 means define `admin-0` as the only bootstrap server and is equivalent to defining `NUODB_BOOTSTRAP_SERVERID=admin-0`, which we do not currently do. 0 means there is no `NUODB_BOOTSTRAP_SERVERID`.

Reply:
what happens if there is no bootstrap server?

Reply:
See my other comment. We have to define `NUODB_DOMAIN_ENTRYPOINT=admin-0` otherwise there is the possibility of disjoint domains being created by concurrently starting admins.

Reply:
My preference would be to stop supporting the broken model of no bootstrapID and hence do not support a model where NUODB_BOOTSTRAP_SERVERID is empty. Hence I would prefer the default of `boostrapServers` to be `1`.

Reply:
Okay, but we have to make it clear to customers that when they upgrade the Helm charts on existing deployments from nuodb-helm-charts <=2.3.1, they have to explicitly set `bootstrapServers=0`.

Reply:
So that it does not generate the new label and hence can be helm upgraded?

Reply:
The issue with an already bootstrapped domain having the `bootstrapServers` value changed, which causes the Admin to complain about its configuration (nuoadmin.conf) not matching its durable state (raftlog).